apps/dc_tools/setup.cfg

```diff
@@ -16,7 +16,7 @@ url = https://github.com/opendatacube/odc-tools/
 include_package_data = true
 zip_safe = false
 packages = find_namespace:
-python_requires = >=3.7
+python_requires = >= 3.9,<=3.10
```

Comment:
At least make it up to and include 3.11. I'd prefer not to have an upper bound on Python version.

Reply:
Upper bound was because of issue with aiohttp reported in https://github.com/opendatacube/datacube-core/issues/1511